pyvista/plotting/plotting.py

```diff
@@ -342,7 +342,7 @@ def import_gltf(self, filename, set_camera=True):
 
         """
         if not _vtk.VTK9:  # pragma: no cover
-            raise RuntimeError('Support for glTF requires VTK v9 or newer')
+            raise VTKVersionError('Support for glTF requires VTK v9 or newer')
```

Comment:
I noticed during the `black` merge that this was missing an import now that the import was moved to line 468. We should import it in this function too, probably right before line 345.